TestModule/TestFunctions.cpp

```diff
@@ -72,7 +72,7 @@ void TestTooManyVolFilesLoaded()
 	auto volsInGameDirectory = CountVolFilesInGameDirectory();
 
 	std::string volPath("./TestModule/TestVolume.vol");
-	for (auto i = volsInGameDirectory; i < 32; i++)
+	for (auto i = volsInGameDirectory; i < 31; i++)
```

Comment:
Actually, as this is test code for the op2ext project, it may make sense to reach in to an internal op2ext header file to grab this value, rather than rely solely on the exported API. Maybe the private constant in VolList.h could be made public? Though there is a difference between the internal buffer size, which must have a terminator entry, and the max number of VOLs that can be added to the `VolList`. Maybe an additional public constant `MaxVolListSize`, perhaps defined in terms of the private constant.

Reply:
Since this code runs in a separate DLL from op2ext, we would have to add it to the public C ABI. I'm not sure this is worth adding for testing purposes. 

This function would be testing GetMaxVolumeCount + 1. 

However, we have not added feedback to the user on how many volumes have been loaded already, so they don't know how close to the limit they are, just what the limit is.